common/changes/@rushstack/heft/user-danade-JestPlugin_2021-05-28-19-52.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft",
+      "comment": "Remove Jest plugin from Heft. To consume the Jest plugin, add @rushstack/heft-jest-plugin as a dependency and include it in heft.json",
+      "type": "major"
+    }
+  ],
```

Comment:
What is the experience like for a person who upgrades to the latest Heft, but did not read the **UPGRADING.md** instructions?  That needs to be an intuitive experience.